src/transformers/models/layoutlmv3/processing_layoutlmv3.py

```diff
@@ -68,26 +100,12 @@ def __init__(self, image_processor=None, tokenizer=None, **kwargs):
 
     def __call__(
         self,
-        images,
+        images: ImageInput,
         text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
         text_pair: Optional[Union[PreTokenizedInput, List[PreTokenizedInput]]] = None,
         boxes: Union[List[List[int]], List[List[List[int]]]] = None,
         word_labels: Optional[Union[List[int], List[List[int]]]] = None,
```

Comment:
Same here, let's put it in kwargs and raise warning when handling BC

Reply:
I just moved these to `text_kwargs` & added backwards compatibility